src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -387,14 +391,15 @@ const DiscoveryActionBar = (props: Props) => {
       fetchWithCreds({ path: `${jobAPIPath}list` }).then(
         (jobsListResponse) => {
           const { status } = jobsListResponse;
-          if (status === 200) {
+          // jobsListResponse will be boilerplate HTML when not logged in
+          if (status === 200 && typeof jobsListResponse.data === 'object') {
```

Comment:
Fixes "uncaught typeError in promise... o.default is undefined" 
see https://healdata.org while not logged in for example of incorrectly parsing a 200 response